importer.go

```diff
@@ -100,7 +101,7 @@ func ImportVariable(ctx context.Context, tf *tfexec.Terraform, client *tfe.Clien
 		return err
 	}
 
-	v, err := fetchVariableByKey(ctx, client, key, ws.ID, 1)
+	v, err := fetchVariableByKey(ctx, client, key, ws.ID, startingPage)
```

Comment:
This is fine as a literal, 0 and 1 are not magic numbers

https://en.wikipedia.org/wiki/Zero_one_infinity_rule

Reply:
```suggestion
	v, err := fetchVariableByKey(ctx, client, key, ws.ID, 1)
```